manim/mobject/geometry.py

```diff
@@ -863,6 +864,225 @@ def __init__(self, n=6, **kwargs):
+    Instead of 3 straight lines connecting the outer points,
+    a Reuleaux triangle has 3 arcs connecting those points,
+    making a shape with constant width.
+
+    Passed arcs are stored as submobjects in the arcpolygon.
+    This means that the arcs are changed along with the arcpolygon,
+    for example when it's shifted, and these arcs can be manipulated
+    after the arcpolygon has been initialized.
+
+    Also both the arcs contained in an ArcPolygonFromArcs, as well as the
+    arcpolygon itself are drawn, which affects draw time in ShowCreation
+    for example. In most cases the arcs themselves don't
+    need to be drawn, in which case they can be passed as invisible.
+
+    .. manim:: ArcPolygonExample
+
+        class ArcPolygonExample(Scene):
+            def construct(self):
+                arc_conf = {"stroke_width":0}
+                poly_conf = {"stroke_width":10,"stroke_color":BLUE,
+                      "fill_opacity":1,"color": PURPLE}
+                a=[-1,0,0]
+                b=[1,0,0]
+                c=[0,np.sqrt(3),0]
+                arc0=ArcBetweenPoints(a,b,radius=2,**arc_conf)
+                arc1=ArcBetweenPoints(b,c,radius=2,**arc_conf)
+                arc2=ArcBetweenPoints(c,a,radius=2,**arc_conf)
+                reuleaux_tri=ArcPolygonFromArcs(arc0,arc1,arc2,**poly_conf)
+                self.play(FadeIn(reuleaux_tri))
+                self.wait(2)
+
+    The ArcPolygon itself can also be hidden so that instead only the contained
+    arcs are drawn. This can be used to easily debug arcs or to highlight them.
+
+    .. manim:: ArcPolygonExample2
+
+        class ArcPolygonExample2(Scene):
+            def construct(self):
+                arc_conf = {"stroke_width":3,"stroke_color":BLUE,
+                    "fill_opacity":0.5,"color": GREEN}
+                poly_conf = {"color":None}
+                a=[-1,0,0]
+                b=[1,0,0]
+                c=[0,np.sqrt(3),0]
+                arc0=ArcBetweenPoints(a,b,radius=2,**arc_conf)
+                arc1=ArcBetweenPoints(b,c,radius=2,**arc_conf)
+                arc2 = ArcBetweenPoints(c, a, radius=2, **{"stroke_color":RED})
+                reuleaux_tri=ArcPolygonFromArcs(arc0,arc1,arc2,**poly_conf)
+                self.play(FadeIn(reuleaux_tri))
+                self.wait(2)
+
+    """
+
+    def __init__(self, *arcs, **kwargs):
+        if not all(isinstance(m, (Arc, ArcBetweenPoints)) for m in arcs):
+            raise ValueError(
+                "All ArcPolygon submobjects must be of type Arc/ArcBetweenPoints"
+            )
+        VMobject.__init__(self, **kwargs)
+        # Adding the arcs like this makes ArcPolygon double as a VGroup.
+        # Also makes changes to the ArcPolygon, such as scaling, affect
+        # the arcs, so that their new values are usable.
+        self.add(*arcs)
+        # This enables the use of ArcPolygon.arcs as a convenience
+        # because ArcPolygon[0] returns itself, not the first Arc.
```

Comment:
These comments make reference to ArcPolygon, but this class is ArcPolygonFromArcs.

1. Does this comment also apply to ArcPolygon? If so, we should copy it inside the code of that class too.
2. Does this comment NOT apply to ArcPolygon? if so, it should refer to ArcPolygonFromArcs and not to ArcPolygon.

Reply:
Good catch there, I wasn't even thinking of transferring my comments to the version @behackl implemented. Adjusted in the next commit.